q2_feature_table/_transform.py

```diff
@@ -21,3 +21,7 @@ def relative_frequency(table: biom.Table) -> biom.Table:
     """
     table.norm(axis='sample', inplace=True)
     return table
+
+def transpose(table: biom.Table) -> biom.Table:
+    table = table.transpose()
```

Comment:
You might want to name this something like `transposed_table` - that way you aren't overwriting your input `table`.